portfolio/src/main/webapp/index.html

```diff
@@ -10,48 +10,46 @@
-            border-radius: 8px;
-            
+
+        div.a{
+            font-size: 20px;
         }
-        horoscope{
-            border-radius: 8px;
-            width: 50%;
-            
+        div.b{
+            font-size: 16px;
         }
     </style>
   </head>
   <body>
     <div id="content">
       <h1><center>Samuel Lellis</center></h1>
-      <p><center>Linkedin: <i>www.linkedin.com/in/samlellis </i> | Github: <i>https://github.com/SamuelLellis</i> </center></p>
-      <p1> I am currently working towards a BS in Computer Science and a BA in Mathematical Sciences at Binghamton University</br>
-      I am not zeroed in on a specific field yet, right now I just wish to explore. </br>
+      <h3> <center> If you have the time to consider giving up, then you have the time to get back to it. </center> </h3> 
+
+      <p><center>Linkedin: <a href = "https://linkedin.com/in/samlellis"><i>www.linkedin.com/in/samlellis </i></a> | Github: <a href = "https://github.com/SamuelLellis"> <i>https://github.com/SamuelLellis</i></a> </center></p>
+
+      <p1><div class = "a"> I am currently working towards a BS in Computer Science and a BA in Mathematical Sciences at Binghamton University</br>
+      I am not zeroed in on a specific field yet, right now I just wish to explore. </br></div>
       </p1>
 
       <h2> <center>Projects </center></h2>
-        <p2><center><b><i>Markov Zodiac Generator</i>: Created the front-end for a Horoscope generator website.</br>
-        <img src = "/images/Horoscope.jpg" style = "width:85%"></br></br> </br>
+        <p2><center><b><i><a href = "http://markovzodiac.xyz/"> Markov Zodiac Generator</i></a>: Created the front-end for a Horoscope generator website.</br>
+        <img src = "/images/Horoscope.jpg" style = "width:65%"></br></br> </br>
         <center><i>Bingbound:</i> Worked on the front-end for a video game</b>
-        <img src = "/images/Bingbound1.png" style = "width:85%"></center>
+        <img src = "/images/Bingbound1.png" style = "width:65%"></center>
         </br> </p2>
 
-
-        <h3> <center> Skills </center> </h3>
-        <p3> Python</br>
-        Java</br>
-        C++</br>
-        C</br>
-        </p3>
-
-
+        <h4> <center> The Road Behind </center> </h4>
+        <p4><div class = "a"> I was born and raised on Long Island. I come from a relatively </br>
+        large family with me being the youngest of 5 siblings. Fighting with my siblings over essentials 
+        such as the last piece of chocolate cake has taught me </br> how cold and hard this world can be be.  </br>
+        I enjoy animals, anime, technology, and making a difference for the better in the lives of people around me. </br>
+         </div>
+        </p4>
 
         <h4> <center> The Road Ahead </center> </h4>
-        <p4> My decision to pursue computer science came in 12th grade after </br>
+        <p4><div class = "a"> My decision to pursue computer science came in 12th grade after </br>
```

Comment:
Try a better name for this div class to make it more meaningful :)

Reply:
Done. Meaningful names allotted!